usso/jwt_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"testing"
 
 	"github.com/juju/usso/openid"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
The build process is giving a gofmt error. It seems to need the imports sorted alphabetically i.e. move the jwt-go import up one line.